events/dvg_venice.txt

```diff
@@ -0,0 +1,1690 @@
+﻿namespace = dvg_ven
+
+# starting patrician event
+dvg_ven.1 = {
+	type = country_event
+	placement = root
+	title = dvg_ven.1.t
+	desc = dvg_ven.1.d
+	flavor = dvg_ven.1.f
+
+	duration = 3
+
+	event_image = {
+		video = "europenorthamerica_rich_and_poor"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/unspecific/europenorthamerica_rich_and_poor"
+
+	icon = "gfx/interface/icons/event_icons/event_portrait.dds"
+
+	trigger = {
+	}
+
+	immediate = {
+	}
+
+	option = {
+		name = dvg_ven.1.a
+		default_option = yes
+
+		add_modifier = {
+			name = dvg_ven_patricians
+			months = -1
+		}
+	}
+}
+
+#Disaster Begins
+dvg_ven.101 = { #nationalist and anti-caste thoughts have coalesced in calls for enfranchisement
```

Comment:
You have to define it in on_actions or a journal to trigger

Reply:
Maybe a journal so you know what is going on. So you can prevent, or make it happen. Now players won't know about this. It could also be a journal that unlocks on these triggers.

Reply:
> Maybe a journal so you know what is going on. So you can prevent, or make it happen. Now players won't know about this. It could also be a journal that unlocks on these triggers.

Sir, you are looking at only half of the commits...

Reply:
There is a journal entry. Please check discord. T.T